api/all.yaml

```diff
@@ -722,6 +748,268 @@ paths:
           content:
             application/json:
               schema: {}
+  /models:
+    post:
+      operationId: createModel
+      summary: |
+        Create a model.
+        WARNING: This API is experimental and will change in future versions.
+      tags:
+        - Models
+      description: |
+        Creates a new model instance.
+        WARNING: This API is experimental and will change in future versions.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateModel'
+      responses:
+        '200':
+          description: The new model was successfully created.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ModelInfo'
```

Comment:
@haogang I see that other endpoints bypass creating dedicated response messages that only contain the info objects even tho DB UC has them.  Will that be a problem with compatability with databricks UC that *does* have things like CreateRegisteredModelResponse?

Reply:
In looking at the DB UC, it looks like those protos were defined before the proto linter was added that required DB protos to break up responses from the request.  So currently, we have a CreateRegisteredModel message that acts as the request and there is a Response message defined inside of it (CreateRegisteredModel.Response) which only has 1 value defined, the RegisteredModelInfo object thats being returned (with that being marked as json_inline).  

The DB handler for this returns a CreateRegisteredModel.Response object and anything that uses that response references the lone value as resp.getRegisteredModelInfo().  I guess my question ultimately is, do we need to keep that definition here where I would basically define a CreateRegisteredModelResponse definition (the same way I did for ListRegisteredModels) that would only have a single info object contained in it (and i'd do that for every API).